services/QuillLMS/engines/comprehension/app/models/comprehension/activity.rb

```diff
@@ -15,7 +15,7 @@ class Activity < ApplicationRecord
     has_many :passages, inverse_of: :activity, dependent: :destroy
     has_many :prompts, inverse_of: :activity, dependent: :destroy
     has_many :turking_rounds, inverse_of: :activity
-    belongs_to :parent_activity, class_name: Comprehension.parent_activity_classname
```

Comment:
We should keep the method call here over the hard-coded `::Activity`

Reply:
So while I generally agree that we should not hard code strings here, this wasn't working as expected when we had it as `Comprehension.parent_activity_classname`. Rails was interpreting it as the `Comprehension::Activity` module class instead of the parent class. I tried a bunch of configurations and the only one that would work was hard coding it as a string as I've done here.

Reply:
Can you write a spec that shows this resolution failure and I will take a looK?